### PR TITLE
Handle wishlist functionality

### LINKED ISSUE #9: Todo: handle wishlist_functionality
The main focus of the issue:

- add the wishlist functionality

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
handled add to cart functionality and total price at the checkout 
Screenshot (114)
Screenshot (115)](https://user-images.githubusercontent.com/46757775/163703750-e7d1bf66-899c-4eac-895d-09d9406dd622.png)
b7ae21b.png)
Screenshot (117)

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
Before push your code remove comment and try to use reducer for operation.
the code overall is good

File: src/App.css [hunk 1]
```diff
@@ -60,4 +60,15 @@
...
+.cart-icon{
+  position: relative;
+}
+.qty-badge-container{
+  position: absolute;
+  top: 1.2rem;
+  right: 0.3rem;
+  width: 1rem;
+  border-radius: 50%;
+  background-color:#e8e4e6;
```
Comment (coldpigli) [inline src/App.css:72]: use variables for color names

File: src/components/CheckoutCard.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            <img src={imageUrl} alt={title} />
+          </div>
+          <div>
+            <div className="card-heading-container">
+              <h3 className="font-md font-medium">{storeName}</h3>
+              <h5 className="font-sm font-light p-b-1">{title}</h5>
+            </div>
+            <span>Quantity: </span>
+            <div className="card-text-container flex-center">
+              {/* <a href="#"><span className="material-icons">add</span></a><input className="slider-text-input" type="text" defaultValue={1} /><a href="#"><span className="material-icons">remove</span></a> */}
```
Comment (coldpigli) [inline src/components/CheckoutCard.jsx:16]: Remove unwanted comment

File: src/components/Header.jsx [hunk 3]
```diff
@@ -1,44 +1,49 @@
...
+  const { cartState } = useCart();
+  const { cartBasket } = cartState;
+  return (
+    <nav className="navigation nav-bar flex-row">
+      <div className="p-1">
+        <Link to="/" className="navbar-brand brand-heading font-lg">
+          Gourmet
+        </Link>
+      </div>
+      <div className="flex-row">
```
Comment (coldpigli) [inline src/components/Header.jsx:15]: Try to use semantic tags often instead of divs

**Review by coldpigli — APPROVED** [review]
Minor suggestions. Everything else looks good

File: src/components/Header.jsx (hunk 3 above)
Comment (vaibhav-mahalle) [reply src/components/Header.jsx:15]: will keep that in mind going forward

File: src/App.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply src/App.css:72]: thanks for suggestion will make required changes once done with basic functionality.

**Pull request merged**